src/all.less

```
@@ -3,4 +3,5 @@
 @import "~esn-frontend-common-libs/src/all.less";
 @import "~esn-frontend-inbox-calendar/src/all.less";
 
+@import "./esn.inbox.libs/app/inbox.libs.less";
```

Please follow the convention for naming:
```suggestion
@import "./esn.inbox.libs/app/app.less";
```